tests/unit/test_java_version_parsing.py

```diff
@@ -109,3 +108,14 @@ def test_guess_mendix_5_java_8_with_missing_java_version(self):
             m2ee_about, runtime_version, stats
         )
         self.assertEqual(8, guessed_version)
+
+    def test_guess_future_mendix_versions_doesnt_error(self):
```

Comment:
I'm not sure what this test case was supposed to do actually. I think I was aiming to trigger the floor operator in `guess_java_version`?